packages/@react-types/button/src/index.d.ts

```diff
@@ -71,6 +71,10 @@ interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
 
 export interface AriaButtonProps<T extends ElementType = 'button'> extends ButtonProps, LinkButtonProps<T>, AriaBaseButtonProps {}
 export interface AriaToggleButtonProps<T extends ElementType = 'button'> extends ToggleButtonProps, AriaBaseButtonProps, AriaButtonElementTypeProps<T> {}
+export interface AriaToggleButtonGroupItemProps<E extends ElementType = 'button'> extends AriaToggleButtonProps<E> {
+  /** An identifier for the item in the `selectedKeys` of a ToggleButtonGroup. */
+  value: Key
```

Comment:
This is really unfortunate. ToggleButton already supports an `id` prop, which gets passed through to the DOM. Overloading that to be a local id matching our other collection components feels weird. So I went with yet another prop name for this, `value`, which sorta matches RadioGroup and CheckboxGroup. But we still have `selectedKeys` at the top-level rather than `value` there. Thoughts?